src/select2/choices.tpl.html

```diff
@@ -1,4 +1,5 @@
-<ul class="ui-select-choices ui-select-choices-content select2-results">
+<ul class="ui-select-choices ui-select-choices-content select2-results"
+    ng-style="{top: position.top+'px', left: position.left+'px'}">
```

Comment:
I'll back that out, it was a leftover from earlier experimentation. The styles are set in the `positionDropdown()` function.